package.json

```diff
@@ -1,9 +1,19 @@
 {
   "devDependencies": {
-    "babel-eslint": "^10.1.0",
+    "@babel/eslint-parser": "^7.19.1",
+    "autoprefixer": "^10.4.13",
     "eslint": "^8.27.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-react": "^7.31.10",
+    "postcss": "^8.4.21",
+    "postcss-import": "^15.1.0",
+    "postcss-nested": "^6.0.0",
     "prettier": "2.7.1"
+  },
+  "dependencies": {
+    "sass": "^1.57.1"
+  },
+  "scripts": {
```

Comment:
this is saying process the tailwind css and output the result into the application.css file. The latter is then processed by postcss.